tree/ntuple/v7/src/RNTupleDescriptor.cxx

```diff
@@ -981,6 +1050,29 @@ void ROOT::Experimental::RNTupleDescriptorBuilder::AddColumn(
    fDescriptor.fColumnDescriptors.emplace(columnId, std::move(c));
 }
 
+
+ROOT::Experimental::RResult<void>
+ROOT::Experimental::RNTupleDescriptorBuilder::AddColumn(RColumnDescriptor &&columnDesc)
+{
+   const auto fieldId = columnDesc.GetFieldId();
+   const auto index = columnDesc.GetIndex();
+
+   if (fDescriptor.fFieldDescriptors.count(fieldId) == 0)
```

Comment:
This test code "seem" to be repeated a few times.  Should/Could it be wrapped into a `CheckValidity` function?
For example maybe:
```
auto validDescriptor = CheckValid( columnDesc ); // or (fieldId, index)
if (!validDescriptor) {
         return R__FORWARD_ERROR(validDescriptor);
```

Reply:
Fixed.